src/components/tooltip/tooltip.module.scss

```diff
@@ -17,42 +17,56 @@ $triangle-color: var(--rp-ui-base-almost-black-light);
 
 .tooltip {
   transition: opacity 300ms ease;
-  margin: $DEFAULT_MARGIN;
+  z-index: 9;
+
   &[data-placement*='bottom'] {
-    padding-top: $ARROW_HEIGHT;
-    z-index: 2;
+    margin-top: $DEFAULT_MARGIN;
   }
 
   &[data-placement*='top'] {
-    padding-bottom: $ARROW_HEIGHT;
+    margin-bottom: $DEFAULT_MARGIN;
+
+    .tooltip-content {
+      transform: translate(0, calc(-100% - 8px));
+    }
   }
 
   &[data-placement*='left'] {
-    padding-right: $ARROW_HEIGHT;
+    padding-right: $ARROW_HEIGHT + $DEFAULT_MARGIN;
+
+    .tooltip-content {
+      transform: translate(0, calc(-50% - $ARROW_LINE));
+    }
   }
 
   &[data-placement*='right'] {
-    padding-left: $ARROW_HEIGHT;
+    margin-left: $DEFAULT_MARGIN;
+
+    .tooltip-content {
+      transform: translate(8px, calc(-50% - $ARROW_LINE));
+    }
   }
 }
 
 .tooltip-content {
+  @include font-scale('s-small');
   padding: 16px;
   border-radius: 8px;
-  background-color: var(--rp-ui-base-almost-black-slight-light);
+  background-color: var(--rp-ui-base-black);
   font-family: var(--rp-ui-base-font-family);
   font-weight: $fw-medium;
-  @include font-scale('s-small');
+  font-size: 11px;
```

Comment:
Don't use `font-size` directly.
Let's follow our predefined set of the sizes for text for consistency.

Reply:
Done.